.ci/cico_compatibility_build_check.sh

```diff
@@ -14,11 +18,11 @@ set -e
 git rebase origin/master || (echo "ERROR: Rebasing branch 'upstream' on top of 'master' failed"; exit 1)
 
 CHE_VERSION=$(curl -s https://raw.githubusercontent.com/eclipse/che/master/pom.xml | grep "^    <version>.*</version>$" | awk -F'[><]' '{print $3}')
-PREV_CHE_VERSION=$(scl enable rh-maven33 'mvn help:evaluate -Dexpression=che.version -q -DforceStdout')
```

Comment:
I believe we do need to continue using the SCL approach

Reply:
up2you. For me, it looks very confusing.